pkgs/development/ocaml-modules/ocamlformat/ocamlformat-lib.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, callPackage, ocaml-ng, version ? "0.25.1" }:
+
+with ocaml-ng.ocamlPackages;
+
+let inherit (callPackage ./generic.nix { inherit version; }) src library_deps;
+
+in assert (lib.versionAtLeast version "0.25.1");
+
+buildDunePackage {
+  pname = "ocamlformat-lib";
+  inherit src version;
+
+  minimalOCamlVersion = "4.08";
+  duneVersion = "3";
+
+  nativeBuildInputs = [ menhir ];
+
+  buildInputs = library_deps;
```

Comment:
I think that this should be propagated build inputs. Right? @vbgl

Reply:
That looks right, thanks!